dist/redhat/build_rpm.sh

```diff
@@ -105,15 +101,11 @@ parameters=(
     -D"version $SCYLLA_VERSION"
     -D"release $SCYLLA_RELEASE"
     -D"package_name $PACKAGE_NAME"
-    -D"cloud_provider $CLOUD_PROVIDER"
     -D"scylla true"
 )
 
 if [[ "$TARGET" = "centos7" ]] || [[ "$TARGET" = "centos8" ]]; then
     rpmbuild "${parameters[@]}" -ba --define '_binary_payload w2.xzdio' --define "_topdir $RPMBUILD" --define "dist .el7" $RPM_JOBS_OPTS $RPMBUILD/SPECS/$PACKAGE_NAME.spec
 else
     rpmbuild "${parameters[@]}" -ba --define '_binary_payload w2.xzdio' --define "_topdir $RPMBUILD" $RPM_JOBS_OPTS $RPMBUILD/SPECS/$PACKAGE_NAME.spec
-fi
-if [[ "$CLOUD_PROVIDER" = "aws" ]]; then
-  cp ${RPMBUILD}/../aws/cloudformation/scylla.yaml $RPMBUILD/CLOUDFORMATION/scylla_cluster_${SCYLLA_VERSION}_${SCYLLA_RELEASE}.yaml
```

Comment:
Why removing cloudformation.yaml ? it's needed for AWS , right?

Reply:
the file `aws/cloudformation/scylla.yaml` is constant and doesn't change

Reply:
It's part of the release, we have a matching on for each release. 

It wasn't changing so far, but it might.

We should leave it in, that what agreed with @tzach and @slivne.

Reply:
@fruch Since this yaml file is something that you don't generating during rpm build process (except adding the scylla version) it doesn't really matter where do you do it. don't mind that it will stay as is, but we can also do it as part of the pipeline

Reply:
@fruch the script is called build_rpm.sh. Its purpose to **build RPM** and not do anything else. 
In addition, per our discussion with Yaron (and as agreed today on the release status meeting with Shlomi and Avi) the script should be cloud provider agnostic, so Yaron will implement this part in the pipeline.